CONTRIBUTING.md

```diff
@@ -169,12 +169,13 @@ You can also run `task -C 1` to limit the number of concurrent tasks, which is u
 
 Finally, since all tests just run `go test` with various arguments and flags under the hood,
 you may also use all standard `go` tool facilities,
-including [`GOFLAGS` environment variable](https://pkg.go.dev/cmd/go#hdr-Environment_variables).
+including [`GOFLAGS` environment variable](https://pkg.go.dev/cmd/go#hdr-Environment_variables)
+or predefined variable `TEST_RUN`.
```

Comment:
The resulting text is simply not correct: `TEST_RUN` is not a "standard `go` tool facility"